src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs

```diff
@@ -13,6 +13,14 @@ public partial class TestServices
 	{
 		public static class KeyboardHelper
 		{
+			static KeyboardHelper()
+			{
+				for (VirtualKey key = VirtualKey.A; key <= VirtualKey.Z; key++)
+				{
+					m_vKeyMapping.Add(char.ToLower((char)key).ToString(), key);
```

Comment:
Should be build based on the current culture? I think this should be cached and rebuilt on demand.

Reply:
What would be the benefit of rebuilding on demand? This is executed once and is test-only code anyway.

Reply:
Ah! My apologies, I missed that it was in tests. It's not needed then.